articles/machine-learning-and-deep-learning/index.md

```diff
@@ -0,0 +1,55 @@
+
+### Machine Learning vs. Deep Learning
+
+Deep learning differs from machine learning in some aspects. One major attribute is that deep learning learns from its mistakes. In most cases, this is achieved without human intervention. As for machine learning, there is a need for a human to correct the mistake made.
+
+Deep learning then adopts an algorithm then learns not to make that mistake again in the future. This enhances the [multi-level neural networks](https://towardsdatascience.com/multi-layer-neural-networks-with-sigmoid-function-deep-learning-for-rookies-2-bf464f09eb7f?gi=e7b1f9430ac5) to learn from their mistakes. But, this model does not guarantee success or no mistakes in the future. This because a wrong prediction can be made depending on the quality of the data fed to them. The accuracy is dependent on the quality of data used to learn.
+
+### Applications and Conclusion
+
+Deep learning is currently being used in various industries and companies. Most of these companies involve the use of the high tech industry in their operations. Besides this, we, as consumers, are currently using these technologies and applications. People have been using and interact with these innovations without even realizing it.
+
+Google applications like Gmail, Google Search, and Google Maps rely on deep learning. This is because such applications need accurate data to improve their intuitiveness. For instance, Gmail can detect important mails for the user. This is determined by the frequency through which a user gets to communicate. This is made effective by using a smart reply that offers a fast response to any email received. This feature depends on the content of the email received.
+
+Besides this, Google Search uses a prediction that anticipates what one is doing. This includes a prediction through past searches, activities, and places visited. The Google Maps application keeps data that can trace the history of past locations. Due to this, the application makes correct and accurate predictions.
+
+Machine learning stimulates the smooth transition of smart applications today. Most of these applications fetch a customized user experience that improves predictability. As a result, the user can enjoy the fast use of services offered by these applications.
+
+Deep learning has increased efficiency thus eliminating the need for human supervision. This has led to increased performance by machines towards performing tasks. This is because deep learning allows the use of an algorithm that automates errors. This automation corrects any mistakes without involving human intervention to correct the error.
+
+Both deep learning and machine learning have promoted efficiency in most applications. As a result, the acceleration and pace of technology has increased.
```

Comment:
Also, mention the economic advantage to companies. Efficiency in terms of time and money is necessary.

Reply:
noted.